File: Core/Entities/Candidate.cs
using System.Collections.Generic;

namespace Core.Entities
{
  public class Candidate
  {
    public int CandidateId { get; set; }
    public string Name { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string SkillTags { get; set; }
    public IList<CandidateSkill> CandidateSkills { get; set; }
  }
}

File: client/src/app/shared/models/candidate.ts
export interface ICandidate {
  candidateId: number;
  name: string;
  firstName: string;
  lastName: string;
  skillTags: string;
  candidateSkills: ICandidateSkill[];
}

export interface ICandidateSkill {
  candidateId: number;
  name: string;
  weighting: number;
}
